### PR TITLE
Added an error page & assets

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
The assets folder is now linked to /assets & the error page is now a catch-all page.

### CODE REVIEW

**Comment by l3gacyb3ta** [comment]
Ah, According to checks, I need to format this..

**Review by veritem — CHANGES_REQUESTED** [review]
Run `yarn format:write` in fronted directory to fix ci

**Comment by l3gacyb3ta** [comment]
Huh, is that the go code? I'll check.

**Comment by l3gacyb3ta** [comment]
YES! I have fixed the code.

**Comment by l3gacyb3ta** [comment]
Wait, there's some stuff I forgot, soooo

File: main.go [hunk 1]
```diff
@@ -46,6 +46,8 @@ func main() {
 
 	r.Use(static.ServeRoot("/", "./frontend/out"))
 
+	r.Use(static.ServeRoot("/assets", "./assets"))
```
Comment (cjdenio) [inline main.go]: Static assets should be served from the frontend rather than the API

**Review by cjdenio — DISMISSED** [review]

File: main.go (hunk 1 above)
Comment (l3gacyb3ta) [reply main.go]: Ah, Ill fix that, that makes sense.

**Comment by anirudhb** [comment]
Looks like this has conflicts now that #95 has been merged. Please rebase and fix those.

**Comment by l3gacyb3ta** [comment]
Ok, I'll run the formaters

**Comment by l3gacyb3ta** [comment]
YAY

**Comment by l3gacyb3ta** [comment]
Eyyy errors merged

**Comment by l3gacyb3ta** [comment]
Ugh formating

**Comment by l3gacyb3ta** [comment]
Okie dokie, everything should be _fine_

**Review by jasonappah — CHANGES_REQUESTED** [review]
lgtm but please remove `frontend/package-lock.json` since we use yarn for dependency management, and also remove the `./hack-as-a-service` binary.

**Comment by l3gacyb3ta** [comment]
Lol sorry for that, didnt mean to commit those.

File: main.go (hunk 1 above)
Comment (veritem) [reply main.go]: Make sure to serve assets on frontend rather than backend

File: frontend/pages/teams/[id].tsx [hunk 2]
```diff
@@ -151,36 +151,6 @@ export default function TeamPage() {
...
-      console.log("[-] Expenses WebSocket is closed");
-    };
-    expensesWs.current.onmessage = (e) => {
-      const expense = parseFloat(JSON.parse(e.data));
-      // console.log(`Raw ${e.data}, parsed ${expense}`);
-      setExpenses((e) => e + expense);
-    };
-
-    return () => expensesWs.current?.close();
-  }, [id]);
```
Comment (jasonappah) [inline frontend/pages/teams/[id].tsx]: Uh can you put this back? I'm guessing this was accidental lol

**Review by jasonappah — CHANGES_REQUESTED** [review]
One more thing I didn't get the first time around, sorry, but it should be good for real this time!

**Review by veritem — DISMISSED** [review]

File: frontend/pages/teams/[id].tsx (hunk 2 above)
Comment (l3gacyb3ta) [reply frontend/pages/teams/[id].tsx]: lol oops

File: main.go (hunk 1 above)
Comment (jasonappah) [reply main.go]: This line should be removed since you've added it to the frontend routes. Fix these last couple things and I'll merge!

**Comment by l3gacyb3ta** [comment]
Please let this fixxx

**Review by jasonappah — APPROVED** [review]
Merging!

**Pull request merged**